Source/Scene/GltfLoader.js

```diff
@@ -482,6 +492,26 @@ function loadVertexAttribute(loader, gltf, accessorId, gltfSemantic, draco) {
 
     attribute.buffer = vertexBufferLoader.vertexBuffer;
 
+    if (
+      attribute.normalized &&
+      gltfSemantic === VertexAttributeSemantic.POSITION
+    ) {
+      var min = AttributeCompression.dequantize(
+        new Float32Array(accessor.min),
+        accessor.componentType,
+        accessor.type,
+        1
+      );
+      var max = AttributeCompression.dequantize(
+        new Float32Array(accessor.max),
+        accessor.componentType,
+        accessor.type,
+        1
+      );
+      attribute.min = Cartesian3.fromArray(min);
+      attribute.max = Cartesian3.fromArray(max);
+    }
```

Comment:
This code isn't being covered: 
![image](https://user-images.githubusercontent.com/8422414/129619830-eff8acef-efd4-48d3-ad68-2eeb2997ad40.png)